contents/articles/2017-01_image_user_auth/index.md

```diff
@@ -0,0 +1,25 @@
+--- 
+title: Personalized Image-based User Authentication using Wearable Cameras
+date: Wed Jan 11 2017 11:18:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Ngu Nguyen et al., propose a novel personalized user authentication mechanism that generates an image-based passwords from egocentric videos captured by wearable cameras..."
```

Comment:
Ngu Nguyen et al., use Pupil to create a personalized user authentication system that generates image based passwords from egocentric video...